HACKING.md

```diff
@@ -2,13 +2,35 @@
 
 To work in the framework itself you will need Python >= 3.8. Linting, testing,
 and docs automation is performed using
-[`tox`](https://tox.readthedocs.io/en/latest/), which you should install.
-For improved performance on the tests, ensure that you have PyYAML
-installed with the correct extensions:
+[`tox`](https://tox.readthedocs.io/en/latest/).
+
+First, make sure to install [uv](https://docs.astral.sh/uv/), for example via:
```

Comment:
We try to avoid "via" -- use "using" instead, or just drop:

```suggestion
First, make sure to install [uv](https://docs.astral.sh/uv/), for example:
```